secure_message/v1/application.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from structlog import wrap_logger
+
+from secure_message.resources.health import Health, DatabaseHealth, HealthDetails
+from secure_message.resources.info import Info
+from secure_message.resources.labels import Labels
+from secure_message.resources.messages import MessageList, MessageSend, MessageById, MessageModifyById
+from secure_message.resources.drafts import DraftSave, DraftById, DraftModifyById, DraftList
+from secure_message.resources.threads import ThreadById, ThreadList
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def set_v1_resources(api):
+    api.add_resource(Health, '/health')
```

Comment:
Are these going to be updated following the tech session review yesterday?

Reply:
These are the v1 resources aren't they?

Reply:
If you mean the health endpoint , I am not aware of any request to change that